scripts/installer.py

```diff
@@ -0,0 +1,161 @@
+    """
+    def setupDNS(self):
+        pass
+
+    """
+    Generates an HTTPS certificate for my.massa using mkcert and stores it in the thyra config folder.
+    """
+    def generateCertificate(self):
+        logging.info("Generating HTTPS certificate for my.massa")
+
+        if not os.path.exists(self.CERTIFICATIONS_FOLDER_PATH):
+            try:
+                os.makedirs(self.CERTIFICATIONS_FOLDER_PATH)
+            except OSError as err:
+                self.printErrorAndExit(f"Error while creating certs folder: {err}")
+
+        self.downloadFile(self.MKCERT_URL, self.MKCERT_FILENAME)
+        os.chmod(self.MKCERT_FILENAME, 0o755)
+
+        self.executeCommand([
+            os.path.join(os.getcwd(), self.MKCERT_FILENAME), 
+            "--install"])
+        self.executeCommand([
+            os.path.join(os.getcwd() , self.MKCERT_FILENAME),
+            "--cert-file", os.path.join(self.CERTIFICATIONS_FOLDER_PATH , "cert.pem"),
+            "--key-file", os.path.join(self.CERTIFICATIONS_FOLDER_PATH, "cert-key.pem"),
+            "my.massa"])
+
+        try:
+            os.remove(self.MKCERT_FILENAME)
+        except OSError as err:
+            self.printErrorAndExit(f"Error while deleting mkcert binary: {err}")
+        logging.info("HTTPS certificate successfully generated")
+
+    """
+    Downloads thyra server and stores it in the thyra install folder.
+    """
+    def installThyraServer(self):
+        self.downloadFile(self.THYRA_SERVER_URL, self.THYRA_SERVER_FILENAME)
+        os.chmod(self.THYRA_SERVER_FILENAME, 0o755)
+        if os.getcwd() != self.THYRA_INSTALL_FOLDER_PATH:
+            try:
+                thyra_server_path = os.path.join(self.THYRA_INSTALL_FOLDER_PATH, self.THYRA_SERVER_FILENAME)
+                if os.path.exists(thyra_server_path):
+                    os.remove(thyra_server_path)
+                shutil.move(self.THYRA_SERVER_FILENAME, self.THYRA_INSTALL_FOLDER_PATH)
+            except OSError as err:
+                self.printErrorAndExit(f"Error while moving thyra server binary: {err}")
+        logging.info("Thyra server installed successfully")
+
+    """
+    Downloads thyra app and stores it in the thyra install folder.
+    """
+    def installThyraApp(self):
+        self.downloadFile(self.THYRA_APP_URL, self.THYRA_APP_FILENAME)
+        os.chmod(self.THYRA_APP_FILENAME, 0o755)
+        if os.getcwd() != self.THYRA_INSTALL_FOLDER_PATH:
+            try:
+                thyra_app_path = os.path.join(self.THYRA_INSTALL_FOLDER_PATH, self.THYRA_APP_FILENAME)
+                if os.path.exists(thyra_app_path):
+                    os.remove(thyra_app_path)
+                shutil.move(self.THYRA_APP_FILENAME, self.THYRA_INSTALL_FOLDER_PATH)
+            except OSError as err:
+                self.printErrorAndExit(f"Error while moving thyra app binary: {err}")
+        logging.info("Thyra app installed successfully")
```

Comment:
looks like a duplication of the previous function.

Reply:
Yes. I thought about making a `installBinary` function that would take, an URL, a binary name and an installation path that would do the same but I didn't did it thinking the Windows install might require some specific things. 
I will add a function to simplify things.